lib/Dialect/TritonGPU/IR/Dialect.cpp

```diff
@@ -1742,20 +1753,19 @@ AMDWmmaEncodingAttr::getShapePerCTATileForDotOperands(ArrayRef<int64_t> shape,
 
 unsigned AMDWmmaEncodingAttr::getTotalElemsPerThreadForOperands(
     ArrayRef<int64_t> shape, Type eltTy, int kWidth, int opIdx) const {
-  auto rep = getWMMARepForOperands(shape, eltTy, kWidth, opIdx);
+  auto rep = getRepForOperands(shape, eltTy, kWidth, opIdx);
   return product(rep) * kWidth;
 }
 
-SmallVector<int64_t>
-AMDWmmaEncodingAttr::getWMMAElemsPerInstrForOperands() const {
+SmallVector<int64_t> AMDWmmaEncodingAttr::getElemsPerInstrForOperands() const {
   return {16, 16};
```

Comment:
Are you sure this is correct? Because I think v1 is 16 but v2 is 8 (I understand what this is).

Reply:
it is relevant for one instruction for all of the threads. One operand is 16x16 submatrix. It is does not matters here how it is mapped to threads.

Reply:
Ok, so this is the operand of the `triton.dot`, I see.